retrofit-converters/simplexml/src/main/java/retrofit/converter/SimpleXMLConverter.java

```diff
@@ -43,35 +43,34 @@ public SimpleXMLConverter(Serializer serializer, boolean strict) {
     this.strict = strict;
   }
 
-  @Override public Object fromBody(TypedInput body, Type type) throws IOException {
+  @Override public Object fromBody(ResponseBody body, Type type) throws IOException {
+    InputStream is = body.byteStream();
     try {
-      return serializer.read((Class<?>) type, body.in(), strict);
+      return serializer.read((Class<?>) type, is, strict);
```

Comment:
Tempting to suggest calling `ResponseBody.charStream()` instead of `ResponseBody.byteStream()`, but it's a potentially-breaking behavior change. XML documents that don't specify the charset in the content-type header can instead specify it in the `<?xml` declaration thingy.

Leave this as-is.